python_modules/dagster/dagster/_core/definitions/asset_dep.py

```diff
@@ -28,6 +28,7 @@ class AssetDep(
         [
             ("asset_key", PublicAttr[AssetKey]),
             ("partition_mapping", PublicAttr[Optional[PartitionMapping]]),
+            ("io_manager_key", PublicAttr[Optional[str]]),
```

Comment:
is this meant to be part of this pr? seems unrelated to fixing the test

Reply:
whoops. removed.